src/generators/app/index.js

```diff
@@ -41,7 +41,17 @@ export const AppMixin = subclass =>
 
     async execute() {
       console.log(header);
-      const scaffoldOptions = [];
+
+      let scaffoldOptions = [];
+
+      // This is needed as when ovverides is informed, it does not load questions and as
```

Comment:
```suggestion
      // This is needed as when overrides is informed, it does not load questions and as
```

Reply:
sorry, my mom's a school teacher - these little nits drive me crazy :D

Reply:
:D No problem, I like these types of comments as it helps me to improve my English !